meta-xt-prod-devel-rcar-control/recipes-guests/domd/files/domd-h3ulcb-4x2g-kf.cfg

```diff
@@ -12,9 +12,9 @@ kernel = "/usr/lib/xen/boot/linux-domd"
 device_tree = "/usr/lib/xen/boot/domd.dtb"
 # Kernel command line options
 # Uncomment this when using nfs as a boot device
-#extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd,vers=3 ip=dhcp rw rootwait console=hvc0 pvrsrvkm.DriverMode=0 xt_page_pool=67108864 xt_cma=100663296"
+#extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd,vers=3 ip=dhcp rw rootwait console=hvc0 pvrsrvkm.DriverMode=0"
```

Comment:
> Please note that we update configs for salvator-x-h3-4x2g
and h3ulcb-4x2g-kf machines only.

What about other machines?

Reply:
Good question, these are the machines for which I have tested the build. I don't have access to other machines. But, I think the change is trivial enough, so I will update for other machines as well.

Reply:
"+"